scss/elements/containers.scss

```
@@ -72,6 +72,21 @@
     }
   }
 
+  &.is-dark {
+    &::before {
+      background-color: $base-color;
+    }
+
+    &::after {
+      border-color: #fff;
+    }
+
+    > .title {
+      color: $background-color;
+      background-color: $base-color !important;
```

I think that `!important` is unnecessary. 🙏 
I don't want to use `!important`. 🙇 
```suggestion
      background-color: $base-color;
```